openapi/tests/openapi_integration/test_delete_points.py

```diff
@@ -38,7 +38,8 @@ def test_delete_points():
         path_params={'collection_name': collection_name},
     )
     assert response.ok
-    assert response.json()['result']['vectors_count'] == 7
+    assert response.json()['result']['points_count'] == 7
+    assert response.json()['result']['vectors_count'] == 8 # We don't propagate deletes to vectors at this time
```

Comment:
How can we know when the vectors are effectively deleted?

Reply:
We don't based on this count at this time.

Our optimizers and scorers also check if the corresponding point is deleted, in which case those vectors are considered to be deleted as well.

Note that explicitly deleting vectors does decrease this count.